test/e2e/example.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect } from '@playwright/test';
+import { test } from './electron-test';
+
+test('basic test', async ({ page }) => {
```

Comment:
We could probably do all of this in cljs. But all examples for playwright tests are in TS, and we are just getting started on them, so it's easier to start with TS.